src/api/index.js

```diff
@@ -71,6 +71,15 @@ const userAboutInfo = (data, resolve, reject) => {
     .then(res => resolve(res))
     .catch(err => reject(err));
 };
+
+const serverDownErrorMessage = (error) => {
+  debugger
+  if (error === 'Failed to fetch') {
```

Comment:
should match with status not string

Reply:
Can only match with string because it only contains string